modules/miniasm/meta.yml

```
@@ -0,0 +1,51 @@
+name: miniasm
+description: A very fast OLC-based de novo assembler for noisy long reads
+keywords:
+  - assembly
+  - pacbio
+  - nanopore
+tools:
+  - miniasm:
+      description: Ultrafast de novo assembly for long noisy reads (though having no consensus step)
+      homepage: https://github.com/lh3/miniasm
+      documentation: https://github.com/lh3/miniasm
+      tool_dev_url: https://github.com/lh3/miniasm
+      doi: ""
```

Found one more point: Please add the citation
```suggestion
      doi: "https://doi.org/10.1093/bioinformatics/btw152"
```